### PR TITLE
move variables to env block

### PR DESCRIPTION
**Pull request by jetersen**

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Comment by jetersen** [comment]
@jglick FYI I don't believe this requires further testing as it was demonstrated to work in interesting categories action.

File: action.yaml [hunk 1]
```diff
@@ -3,10 +3,13 @@ description: Deploys a Maven project to the Jenkins Artifactory repository in co
 inputs:
   GITHUB_TOKEN:
     required: true
+    description: Token to run with, defaults to the repository GITHUB_TOKEN
   MAVEN_USERNAME:
     required: true
+    description: Maven Username used to release to Jenkins Artifactory repository
   MAVEN_TOKEN:
```
Comment (jetersen) [inline action.yaml:10]: These really ought to be moved to secrets section.

**Review by jglick — APPROVED** [review]
Looks right. Would be prudent to test in case there is a typo or whatever.

File: action.yaml [hunk 2]
```diff
@@ -3,10 +3,13 @@ description: Deploys a Maven project to the Jenkins Artifactory repository in co
 inputs:
   GITHUB_TOKEN:
     required: true
+    description: Token to run with, defaults to the repository GITHUB_TOKEN
   MAVEN_USERNAME:
     required: true
+    description: Maven Username used to release to Jenkins Artifactory repository
```
Comment (jetersen) [inline action.yaml]:
```suggestion
    description: Maven username used for deploying the plugin jar to Jenkins Artifactory Repository
```

File: action.yaml [hunk 3]
```diff
@@ -3,10 +3,13 @@ description: Deploys a Maven project to the Jenkins Artifactory repository in co
...
   MAVEN_TOKEN:
     required: true
+    description: Maven Token used to release to Jenkins Artifactory repository
```
Comment (jetersen) [inline action.yaml]:
```suggestion
    description: Maven token used for deploying the plugin jar to Jenkins Artifactory Repository
```

**Review by jetersen — COMMENTED** [review]
Use similar wording as in https://github.com/jenkins-infra/github-reusable-workflows/blob/main/.github/workflows/maven-cd.yml

**Comment by jetersen** [comment]
@jglick Seems to be working as intended:

https://github.com/jenkinsci/jenkins-infra-test-plugin/runs/6871373890?check_suite_focus=true#step:4:85

**Pull request merged**